src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -110,24 +110,6 @@ namespace WaterToAirHeatPumpSimple {
     // State Energy Simulation Program. M.S. Thesis, Department of Mechanical and Aerospace Engineering,
     // Oklahoma State University. (downloadable from www.hvac.okstate.edu)
 
-    // OTHER NOTES:
-
-    // USE STATEMENTS:
-    // Use statements for data only modules
-    // Use statements for access to subroutines in other modules
-    // Using/Aliasing
-    using namespace DataLoopNode;
-    using namespace DataSizing;
-    using DataHVACGlobals::ContFanCycCoil;
-    using DataHVACGlobals::Cooling;
-    using DataHVACGlobals::CycFanCycCoil;
-    using DataHVACGlobals::Heating;
-    using DataHVACGlobals::WaterConstant;
-    using DataHVACGlobals::WaterConstantOnDemand;
-    using DataHVACGlobals::WaterCycling;
-    using DataPlant::TypeOf_CoilWAHPCoolingEquationFit;
-    using DataPlant::TypeOf_CoilWAHPHeatingEquationFit;
```

Comment:
Thanks for cleaning these up.

Reply:
I did the clean up for this coil so I'll do the walk-thru.